fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceGroup.java

```diff
@@ -219,6 +219,7 @@ public void addResources(List<IDatabaseObject> group, String prefix) {
                 .addCharColumn(           IS_DELETED,           1, false)
                 .addBlobColumn(                 DATA,  2147483647,  10240,   true)
                 .addUniqueIndex(tableName + "_PRF_IN1", prfIndexCols, prfIncludeCols)
+                .addIndex(IDX + tableName + LOGICAL_RESOURCE_ID, LOGICAL_RESOURCE_ID)
                 .addPrimaryKey(tableName + "_PK", RESOURCE_ID)
```

Comment:
Question: 
is RESOURCE_ID ever used on it's own? 
you may be able to change the index into the primary key and replace RESOURCE_ID as the PK

Reply:
yeah, resource_id is used separately also.

Reply:
makes sense